openvdb/io/File.cc

```diff
@@ -71,10 +71,20 @@ struct File::Impl
     static Index64 getDefaultCopyMaxBytes()
     {
         Index64 result = DEFAULT_COPY_MAX_BYTES;
+#if defined(_MSC_VER)
+        char* s;
+        _dupenv_s(&s, nullptr, "OPENVDB_DISABLE_DELAYED_LOAD");
```

Comment:
Wrong env var here - should be OPENVDB_DELAYED_LOAD_COPY_MAX_BYTES

Reply:
Yikes thanks